selftests/unit/test_nrunner.py

```diff
@@ -92,14 +92,14 @@ class RunnableFromCommandLineArgs(unittest.TestCase):
 
     def test_noop(self):
         parsed_args = {'kind': 'noop', 'uri': None}
-        runnable = nrunner.runnable_from_args(parsed_args)
+        runnable = nrunner.Runnable.from_args(parsed_args)
```

Comment:
This (and all other instances of similar changes here) deserves to be part of 69ab640b1d79974971c073518a1197d20f50ee9d.